### PR TITLE
Fix sanity issues raised from testing in Ansible 2.10

### PR DESCRIPTION
**Pull request by parkrish**
Fix sanity errors seen ,when testing with Ansible 2.10

### CODE REVIEW

File: requirements.test.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ck==19.3b0 ; python_version > '3.5'
```
Comment (gundalow) [inline requirements.test.txt:1]: What uses this flie?

Comment (parkrish) [reply requirements.test.txt:1]: @gundalow , We added this file to list the packages to be pre-installed to run the sanity test for this collection. This should help developers run sanity test before submitting any change. Do you think it is not necessary ?

Comment (gundalow) [reply requirements.test.txt:1]: If you run `ansible-test sanity --venv` or
`ansible-test sanity --docker` the requirements file under test/sanity will be automatically installed. 

See community.general for an example https://github.com/ansible-collections/community.general/blob/master/tests/unit/requirements.txt

Comment (gundalow) [reply requirements.test.txt:1]: So when you add `5` from https://github.com/ansible-collections/dellemc.os10/issues/20 you will have CI triggered on every PR to this repo.

Comment (parkrish) [reply requirements.test.txt]: @gundalow ,thanks for the clarification.I have removed the requirements.test.txt now and have added the requiremtns.txt under tests/sanity directory. Now, I get error no module named "ansible.netcommon". How do we have ansible.netcommon installed,when running automated sanity tests for pull-requests ?

Comment (gundalow) [reply requirements.test.txt]: `ansible-galaxy collection install ansible.netcommon -p ../../`

Comment (parkrish) [reply requirements.test.txt]: Added "ansible-galaxy collection install ansible.netcommon -p ../../" to the tests/sanity/requirements.txt, but it did not help, still getting the same error

Comment (gundalow) [reply requirements.test.txt]: Apologies, this should be a new step in the workflow after ansible has been installed.

Comment (parkrish) [reply requirements.test.txt]: @gundalow , to clarify, when I install netcommon and run "ansible-test sanity --docker" locally, there are no issues.
The error happens only for the sanity test triggered automatically on the Pull-requests in github. 
Is there anyway , I can force installation of netcommon, before the automatic test run ?

File: .github/workflows/ansible-test.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+          path: ansible_collections/dellemc/os10
+
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+
+      - name: Install ansible-base (${{ matrix.ansible }})
+        run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
+
```
Comment (gundalow) [inline .github/workflows/ansible-test.yml:28]: Add a new section here which does ansible-galaxy collection install ansible.netcommon -p ../..

**Comment by javeedf** [comment]
@gundalow is there a way we can generate docs from the modules like os_config.py or os_command.py?

similar to https://github.com/ansible-collections/cisco.ios/blob/master/docs/cisco.ios.ios_acl_interfaces_module.rst

File: .github/workflows/ansible-test.yml (hunk 2 above)
Comment (parkrish) [reply .github/workflows/ansible-test.yml:28]: Thanks @gundalow  for the continuous support. WIth this suggestion, the sanity test ran fine.

File: plugins/modules/os10_command.py [hunk 3]
```diff
@@ -17,7 +17,7 @@
 DOCUMENTATION = """
 ---
 module: os10_command
-version_added: "2.2"
+version_added: "2.9.6"
```
Comment (gundalow) [inline plugins/modules/os10_command.py:20]: You can remove all the `version_added:` strings for all modules and all module's options.

File: requirements.test.txt (hunk 1 above)
Comment (parkrish) [reply requirements.test.txt]: After adding relevant details to the ansible-test.yml, this works fine.Thanks.

**Comment by gundalow** [comment]
Great work everybody.
Can you please replicate this on `dellems.os6`  and `dellems.os9` ?

**Comment by gundalow** [comment]
> @gundalow is there a way we can generate docs from the modules like os_config.py or os_command.py?
> 
> similar to https://github.com/ansible-collections/cisco.ios/blob/master/docs/cisco.ios.ios_acl_interfaces_module.rst

For these three `dellemc.osX` the docs will be generated automatically and hosted on docs.ansible.com. This is being worked on at the moment. Please subscribe (button on right hand side) to https://github.com/ansible-collections/overview/issues/45 for updates

**Comment by javeedf** [comment]
@gundalow Thank you, 
we are also developing 1 more collection,  could you please let us know how this cisco.ios.ios_acl_interfaces_module.rst is generated from module docs? any tools for that?

https://github.com/ansible-collections/cisco.ios/blob/master/docs/cisco.ios.ios_acl_interfaces_module.rst

**Comment by javeedf** [comment]
> Great work everybody.
> Can you please replicate this on `dellemc.os6` and `dellemc.os9` ?

Sure @gundalow

**Pull request merged**